packages/lxd/test/file/push.coffee

```diff
@@ -56,51 +56,64 @@ describe 'lxc.file.push', ->
         $ssh: ssh
         $tmpdir: true
       , ({metadata: {tmpdir}, registry}) ->
-        registry.register 'clean', ->
-          @lxc.delete 'nikita-file-push-2', force: true
-        await @clean()
-        await @lxc.init
-          image: "images:#{images.alpine}"
-          container: 'nikita-file-push-2'
-          start: true
-        await @lxc.exec
-          $$: retry: 3, sleep: 200 # Wait for network to be ready
-          container: 'nikita-file-push-2'
-          command: 'apk add openssl'
-        await @file
-          target: "#{tmpdir}/a_file"
-          content: 'something'
-        {$status} = await @lxc.file.push
-          container: 'nikita-file-push-2'
-          source: "#{tmpdir}/a_file"
-          target: '/root/a_file'
-        $status.should.be.true()
-        {$status} = await @lxc.file.exists
-          container: 'nikita-file-push-2'
-          target: '/root/a_file'
-        $status.should.be.true()
-        await @clean()
+        cluster =
```

Comment:
We do not use cluster to test push, rephrased, we do not introduce a first component with a second component which depends on the first component.

Reply:
Done.